From e025a03fdc8deb2e8cfdcc78706a131955a661d9 Mon Sep 17 00:00:00 2001
From: xiaoyao <xiaoyao@rock-chips.com>
Date: Sat, 8 Oct 2016 11:39:21 +0800
Subject: [PATCH] ARM64: dts: rk3399-box: add card-detect-delay property

Practice shows :
	The sd cards are easier to be identified after increase delay

Change-Id: I48912e2d184902fab8b27edba70281f0bf19b9ab
Signed-off-by: xiaoyao <xiaoyao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index 11443aa59887..0d77c8192b61 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -225,6 +225,7 @@
 	vqmmc-supply = <&vcc_sd>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&sdmmc_clk &sdmmc_cmd &sdmmc_cd &sdmmc_bus4>;
+	card-detect-delay = <800>;
 	status = "okay";
 };
 
-- 
2.35.3

